src/AccessLogEntry.h

```diff
@@ -184,6 +184,11 @@ class AccessLogEntry: public RefCountable
     /// key=value pairs returned from URL rewrite/redirect helper
     NotePairs::Pointer notes;
 
+    /// The number of primary request attempts, done so far.
```

Comment:
Instead of duplicating logformat code documentation here, I would refer to it:

```
/// The number of request sending attempts. See %request_attempts.
```

This variant also avoids the incorrect implication that most other ALE fields are not about the "primary" messages.